### ISSUE TITLE
Improve contrast on patient experience pages

### ISSUE THREAD

**Issue by BobanL** [CDCgov/prime-simplereport#4460]
## Area or Component

PXP - TOS page 
PXP - DOB page

## Quick Summary

Contrast between the elements and the background is low which impacts readability

## Accessibility Violation

[All elements must have sufficient contrast](https://dequeuniversity.com/rules/axe/4.4/color-contrast-enhanced)

## Action Requested

Either make the background white across all the PXP pages or place content inside a white container

## Source Code (optional)

[which file or line is the issue attached to?]: #
